docs/api.rst

```diff
@@ -0,0 +1,6 @@
+API documentation
+~~~~~~~~~~~~~~~~~
+
+.. automodule:: pytket.extensions.pennylane
```

Comment:
Does this include the `pennylane_to_tk` function?

Reply:
No it doesn't, I wasn't sure if it was required to be public (or just used by `PytketDevice.apply()`), but can easily add it.

Reply:
yes please, sorry there should have been a docstring on there